server/templates/browser/kg_main.html

```diff
@@ -0,0 +1,123 @@
+{#
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+#}
+
+{% extends 'base.html' %}
+
+{% block head %}
+  <title>Data Commons KG</title>
+  <link rel="stylesheet" href={{url_for('static', filename='css/homepage.css', t=config['GAE_VERSION'])}}>
+{% endblock %}
+
+{% block content %}
+  <div id="homepage-content">
+    <p>
+      The Data Commons Graph (DCG) is constructed by synthesizing a single Graph from many different data sources.
+      It currently contains data from Wikipedia, the US Census, NOAA, FBI, etc.
+      The browser can be used both to explore what data is available and to understand the graph structure, to help use the API.
+    </p>
+    <p>
+      You can use the search box above or start looking around from some well known starting points:
+    </p>
+    <ul>
+      <li class="type-list">
```

Comment:
Add a todo to pass a json from the route to here and use for loop for these lists?

Reply:
Done. Will probably keep the config in python